test/new_relic/agent/configuration/event_harvest_config_test.rb

```diff
@@ -88,5 +88,23 @@ def test_to_config_hash_with_omitted_event_type
       }
       assert_equal expected, EventHarvestConfig.to_config_hash(connect_reply)
     end
+
+    def test_to_config_hash_with_zero_response_for_log_event_data
+      connect_reply = {
+        'event_harvest_config' => {
+          'report_period_ms' => 5000,
+          'harvest_limits' => {
+            'log_event_data' => 0
+          }
+        }
+      }
+
+      expected = {
+        :'application_logging.forwarding.max_samples_stored' => 0,
+        :'event_report_period.log_event_data' => 5,
+        :event_report_period => 5
+      }
+      assert_equal expected, EventHarvestConfig.to_config_hash(connect_reply)
+    end
```

Comment:
@tannalynn  - we chatted a bit about how this test would work last week. Do you think it's in the right spot?

Reply:
Yeah, this seems good to me! I do think this is the right place for it

Reply:
Thanks!